languages/JavaScript/exercises/concept/arrays-basic/.docs/hints.md

```diff
@@ -0,0 +1,30 @@
+### 1. Retrieve a card from a stack
+
+- Note that the array index starts from `0`. [Here][access_array_elements_resource] you can find more information about how to access array elements.
+
+### 2. Exchange a card in the stack
+
+- The array is a mutable structure, you can change its content anytime.
+
+### 3. Create a stack of cards
+
+- The `push` method adds a new value at the end of the array. [Here][push_method_docs] you can find documentation and an example for the `push` method.
```

Comment:
It would be really nice not to give away the actual solution here. Maybe do:

```suggestion
- There is a [built-in][push_method_docs] method to add a new value to the end of the array.
```

So the reason we don't want to explain the solution directly and instead refer to external documentation, is that we then introduce the student to the fact that there is external documentation and that becomes a starting point for the student to know where to look for when wanting to try and see what other methods there are.

Reply:
This approach makes sense!